API-Samples/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 2.8.11)
 if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
     add_definitions(-DVK_USE_PLATFORM_WIN32_KHR -DWIN32_LEAN_AND_MEAN)
     set(DisplayServer Win32)
-elseif(CMAKE_SYSTEM_NAME STREQUAL "Android")
+elseif(CMAKE_SYSTEM_NAME STREQUAL "Darwin" AND ANDROID)
```

Comment:
I think you want to enable VK_USE_PLATFORM_ANDROID_KHR on any host OS that is targeting Android, not just Darwin.  Perhaps you could add an empty block here for Darwin, and add a new block below that checks for ANDROID on any OS.